README.md

```diff
@@ -0,0 +1,55 @@
+# in-toto Layout Creation Wizard
+
+A Flask based web app to guide project owners through creating an
+[in-toto layout](https://in-toto.io).
+
+Mockups can be found at [`editor-and-wizard-wip/mockups`](https://github.com/in-toto/layout-web-tool/blob/editor-and-wizard-wip/mockups/layout-wizard.pdf).
+
+
+### Installation
+
+**Requirements**
+- [Python 2.7](https://www.python.org/download/releases/2.7/) --  backend
```

Comment:
Do you have to install any Python development headers?

Reply:
Done